internal/bootstrap/git/git.go

```diff
@@ -34,8 +34,10 @@ type Author struct {
 
 type Commit struct {
 	Author
-	Hash    string
-	Message string
+	Hash          string
+	Message       string
+	GpgKeyPath    string
+	GpgPassphrase string
```

Comment:
Think this should not be added to this structure, as it is kind of pre-flight data which only has a purpose while a commit is made, and contains data you generally do not want to keep around for too long because of their sensitive nature.